.changeset/seven-nails-wave.md

```diff
@@ -0,0 +1,19 @@
+---
+'@spectrum-web-components/button': patch
+'@spectrum-web-components/reactive-controllers': patch
+---
+
+**Fixed** aria-label handling in button components and pending state controller.
+
+**Button component changes:**
+
+- Fixed timing of aria-label updates to occur after slot content changes are processed
+- Added `label` property support for programmatic aria-label control
```

Comment:
Is this accurate?

Reply:
no thank you for checking, i need to update this with the new changes